src/components/browser/index.js

```diff
@@ -0,0 +1,217 @@
+// Copyright 2014 Globo.com Player authors. All rights reserved.
```

Comment:
It would be better to keep the `index.js` file just as an entry point to the module.
I suggest creating a `browser.js` file with this code, and `index.js` just load and export the `browser.js` content

Reply:
Also, we can take advantage to remove these old comment header :)